runtime-sdk/src/types/transaction.rs

```diff
@@ -148,6 +148,9 @@ pub enum CallResult {
 
         #[serde(rename = "code")]
         code: u32,
+
+        #[serde(rename = "message")]
+        message: String,
```

Comment:
Make it optional using `#[serde(skip_serializing_if = "String::is_empty")]` and `#[serde(default)]`.

Reply:
ok

Reply:
I should say: these often won't be empty in our usage, with (all?) errors in this project having non-empty displays